dask_cuda/explicit_comms/dataframe_merge.py

```diff
@@ -139,7 +139,7 @@ async def single_partition_join(
     return left_table.merge(right_table, left_on=left_on, right_on=right_on)
 
 
-async def _dataframe_merge(s, dfs_nparts, dfs_parts, left_on, right_on):
+async def _dataframe_merge(s, world, dfs_nparts, dfs_parts, left_on, right_on):
```

Comment:
What is `world` in this context? The pool of all endpoints? Can we make that name a bit more obvious to its usage?

Reply:
Calling it `workers` now